LayoutTests/css2.1/t1202-counters-09-b-expected.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>Test reference: content: counters(c, &quot;.", georgian)</title>
+  <meta charset="utf-8">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#propdef-content">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/syndata.html#counter">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#counter-styles">
+  <style>
+  body { white-space: nowrap; overflow: hidden; }
```

Comment:
The test file doesn't have `overflow: hidden` hence the scrollbar difference. You can either add it to the test file or remove it from here.

Reply:
Thanks!